.github/workflows/production.yml

```diff
@@ -57,3 +57,17 @@ jobs:
           K8S_SECRET_DATABASE_DB: ${{ secrets.K8S_SECRET_DATABASE_DB_STABLE }}
           K8S_SECRET_DATABASE_USERNAME: ${{ secrets.K8S_SECRET_DATABASE_USERNAME_STABLE }}
           K8S_SECRET_DATABASE_PASSWORD: ${{ secrets.K8S_SECRET_DATABASE_PASSWORD_STABLE }}
+
+      - name: Update GTFS feeds
+        uses: City-of-Helsinki/setup-cronjob-action@main
+        with:
+          name: gtfs-feed-update-cronjob
+          image_repository: ghcr.io/city-of-helsinki/${{ github.event.repository.name }}
+          image_tag: ${{ github.sha }}
+          kubeconfig_raw: ${{ env.KUBECONFIG_RAW }} # OR ${{ secrets.KUBECONFIG_RAW_STABLE }} ?
```

Comment:
@Hi-Fi I wonder if this value is correct for production and staging?

Reply:
`KUBECONFIG_RAW_STABLE`